src/main/java/com/melon/portfoliomanager/services/KafkaService.java

```diff
@@ -0,0 +1,24 @@
+package com.melon.portfoliomanager.services;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service("kafka")
+public class KafkaService implements MessageBrokerService {
```

Comment:
How would we support more than 1 topic?

Reply:
I guess we can create the topics we need inside ```KafkaTopicConfig``` class and then autowire the beans that we need.
Is there a better way to do it ?

Reply:
In the current implementation if we add a 2nd topic it seems you would need a 2nd KafkaService. Having >1 topic is a very reasonable expectation and we should support it from the start, even if we don't currently have 2 topics.

Reply:
I do not see the reason why can not we just name our beans and autowire them in one KafkaService. What bothers you being implemented this way ? (there is no need to name the bean in the moment and because of this I have not named it)